opm/parser/share/keywords/000_Eclipse100/W/WCONPROD

```diff
@@ -7,7 +7,7 @@
        {"name" : "GRAT"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "GasSurfaceVolume/Time"},
        {"name" : "LRAT"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
        {"name" : "RESV"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
-       {"name" : "BHP"          , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "Pressure"},
+       {"name" : "BHP"          , "value_type" : "DOUBLE" , "default" : 1.01325 , "dimension" : "Pressure"},
```

Comment:
Three questions here for @joakim-hove mostly:
 - What units should default values use in the keyword?
 - Is there a way to use the unit constants from the C++ code here somehow? (I guess not.)
 - Is there a way to put comments in these files? JSON seems to not support it, so I guess it's a no, too.